src/main/java/com/resourcefulbees/resourcefulbees/utils/validation/ValidatorUtils.java

```diff
@@ -5,7 +5,7 @@
 import static com.resourcefulbees.resourcefulbees.ResourcefulBees.LOGGER;
 
 public class ValidatorUtils {
-    public static final Pattern SINGLE_RESOURCE_PATTERN = Pattern.compile("^(\\w+):(\\w+)$", Pattern.CASE_INSENSITIVE);
+    public static final Pattern SINGLE_RESOURCE_PATTERN = Pattern.compile("^([\\w,\\-,\\.]+):([\\w,\\-,\\.,\\/]+)$", Pattern.CASE_INSENSITIVE);
```

Comment:
You may have wanted to leave the commas out?
```suggestion
    public static final Pattern SINGLE_RESOURCE_PATTERN = Pattern.compile("^([\\w\\-\\.]+):([\\w\\-\\.\\/]+)$", Pattern.CASE_INSENSITIVE);
```

Reply:
You are absolutely right, my mistake!